multisig/proposals/ethereum/vip-049/index.ts

```diff
@@ -0,0 +1,27 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { makeProposal } from "../../../../src/utils";
+
+export const XVS_VAULT_TREASURY = "0xaE39C38AF957338b3cEE2b3E5d825ea88df02EfE";
+export const XVS_STORE = "0x1Db646E1Ab05571AF99e47e8F909801e5C99d37B";
```

Comment:
We don't need this. We should use the function `fundXVSVault` instead of `sweepToken`. The function `fundXVSVault` transfers the funds directly to the XVS Store contract associated with the configured XVS Vault